spec/factories.rb

```diff
@@ -1,2 +1,6 @@
-FactoryBot.define do  
+FactoryBot.define do
+  factory :family do
+    last_name "Liso"
+    capability 3
```

Comment:
👍 

Can the capability ever be zero?

Reply:
UM, no in that case I suppose it would be 'inactive'. Maybe I wil check with my mum. Thanks!